src/Cello/mesh_Block.cpp

```diff
@@ -102,6 +102,10 @@ Block::Block ( MsgRefine * msg )
 
   if (is_first_cycle) {
     apply_initial_();
+
+    if (this->level() > 0){
+      msg->update(data());
+    }
```

Comment:
In my test I only saw calls to p_set_msg_refine. The particle ICs worked without the inclusion of this if statement in Block::Block. However, the similarity of these two functions made me think it should go here as well.

Reply:
@jobordner would you mind commenting on this?